hospital/apphospital/models.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import datetime
+from django.db import models
+
+
+class Hospital(models.Model):
+    hospital_id = models.AutoField(primary_key=True)
+    hospital_name = models.CharField(max_length=200)
```

Comment:
remove the prefix `hospital` from the data memebers.
We don't add the name of the call as prefix

Reply:
d